File: bank.cpp
#include <iostream>
using namespace std;

int main() {
    cout << "~~~~~~~~~~~~ BANK OF TRITON ~~~~~~~~~~~~ " << "\n";
    string accName;
    int password;
    cout << "Enter your account information" << "\n";
    cout << "Username: ";
    cin >> accName;
    cout << "Password: ";
    cin >> password;
    
    while (password != 124816) {
        cout << "Invalid Password" << "\n";
        cout << "Password: ";
        cin >> password;
        if (password == 124816) {
            cout << "You have successfully logged in!";
            break;
        } 
    }
}

File: bank.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});
console.log("~~~~~~~~~~~~~ BANK OF TRITON ~~~~~~~~~~~~~")
console.log("Enter your account information\n")
rl.question("Username: ", (name) => {
    const getPswd = () => rl.question("Password: ", (pswd) => {
        if (pswd !== "124816") {
          console.log("Invalid password")
           getPswd()
        } else {
          console.log("Successfully logged in!")
        }
        rl.close();
    });
    getPswd()
});
